### PR TITLE
API endpoint for reading random numbers from the witness

### PR DESCRIPTION
**Pull request by obucina**
Test cmd:
curl --data '{"method": "call", "params": ["database", "get_random_number", [1000]], "jsonrpc": "2.0", "id": 1}' http://127.0.0.1:8090/ws
curl --data '{"method": "call", "params": ["database", "get_random_number", [5000]], "jsonrpc": "2.0", "id": 1}' http://127.0.0.1:8090/ws
curl --data '{"method": "call", "params": ["database", "get_random_number", [1000000]], "jsonrpc": "2.0", "id": 1}' http://127.0.0.1:8090/ws

### CODE REVIEW

**Comment by obucinac** [comment]
Replaces https://github.com/peerplays-network/peerplays/pull/56

File: libraries/app/include/graphene/app/database_api.hpp [hunk 1]
```diff
@@ -667,6 +667,14 @@ class database_api
        */
       vector<tournament_id_type> get_registered_tournaments(account_id_type account_filter, uint32_t limit) const;
 
+      //////////
+      // RNG //
+      //////////
+      /**
+       * @return Current Random number
```
Comment (oxarbitrage) [inline libraries/app/include/graphene/app/database_api.hpp]: I think we can add the `bound` argument with a brief description above here. Also maybe change `RNG` with `Random Number Generator`, is a bit confusing on the first sight.

**Comment by oxarbitrage** [comment]
By the way, we have the declaration of `get_random_bits` is in the db_getter section: https://github.com/peerplays-network/peerplays/blob/develop/libraries/chain/include/graphene/chain/database.hpp#L279 while the implementation is in db_update: https://github.com/peerplays-network/peerplays/blob/develop/libraries/chain/db_update.cpp#L546

I think we can change one or the other.

**Comment by bobinson** [comment]
Rate limiting idea https://github.com/bitshares/bitshares-core/issues/843

**Pull request merged**